Add instructions for external contributions and code of conduct (#62)
dfinity/stable-structures: Contributing.md

# Contributing

Thank you for your interest in contributing to this repo.
By participating in this project, you agree to abide by our [Code of Conduct](./CODE_OF_CONDUCT.md).

As a member of the community, you are invited and encouraged to contribute by submitting issues, offering suggestions for improvements, adding review comments to existing pull requests, or creating new pull requests to fix issues.

All contributions to DFINITY documentation and the developer community are respected and appreciated.
Your participation is an important factor in the success of the Internet Computer.

## Contents of this repository

This repository contains a collection of scalable data structures for the Internet Computer that persist across upgrades.

## Before you contribute

Before contributing, please take a few minutes to review these contributor guidelines.
The contributor guidelines are intended to make the contribution process easy and effective for everyone involved in addressing your issue, assessing changes, and finalizing your pull requests.

Before contributing, consider the following:

- If you want to report an issue, click **Issues**.

> - If you have more general questions related to Candid and its use, post a message to the [community forum](https://forum.dfinity.org/) or submit a [support request](mailto://support@dfinity.org).

Reviewer: Is this supposed to say Candid?

Author: Fixed.

- If you are reporting a bug, provide as much information about the problem as possible.

- If you want to contribute directly to this repository, typical fixes might include any of the following:

    - Fixes to resolve bugs or documentation errors
    - Code improvements

>     - Feature requests

Reviewer 2: This is a strange paragraph.

Author: Do you have a suggestion on how to improve it?

Reviewer 3: nit: In the sentence "..typical fixes might include any of the following" I will not use "fixes" because not all enumerated things are fixes. Maybe just change the sentence to `You may contribute by:`

> - If you want to contribute directly to this repository, typical fixes might include any of the following:
> 
>     - Fixes to resolve bugs or documentation errors
>     - Code improvements
>     - Feature requests

Reviewer 2: ```suggestion
- You may also contribute directly to the repository. Typical contributions include:

    - Fixes to resolve bugs or documentation errors
    - Code improvements
    - Feature requests
```
